mathics/builtin/box/graphics3d.py

```diff
@@ -303,12 +303,6 @@ def calc_dimensions(final_pass=True):
                             light["target"][j] * boxscale[j] for j in range(3)
                         ]
 
-                # Rescale viewpoint
```

Comment:
Recall Asymptote is another backend. If boxing is changed, that too needs to be adjusted. 

Better would be some way for the backend to specify its preferences and do this conditionally.

Reply:
> Recall Asymptote is another backend. If boxing is changed, that too needs to be adjusted.

I already fixed it.

> Better would be some way for the backend to specify its preferences and do this conditionally.

For now, I moved this code to another place.